src/server/pegasus_server_impl.cpp

```diff
@@ -2113,6 +2139,62 @@ void pegasus_server_impl::update_server_rocksdb_statistics()
     }
 }
 
+void pegasus_server_impl::update_server_rocksdb_rate_limiter_statistics(
+    std::chrono::seconds update_rdb_stat_short_interval)
+{
+    // Update _pfc_rdb_write_limiter_rate_bytes
+    if (_s_rate_limiter) {
```

Comment:
You can disable _update_server_rdb_limiter_stat at the beginning, no need to judge _s_rate_limiter every time.